design/local-node-agent.md

```diff
@@ -45,44 +45,36 @@ The agent will wait for the driver to complete the handshake before moving on an
 Note that the Rook operator will continue to run its existing [dynamic volume provisioner](https://github.com/rook/rook/tree/master/pkg/operator/provisioner) to provision and delete persistent volumes as needed.
 
 ### **Flexvolume**
-#### Control Flow Overview
-The Rook Flexvolume driver will be very lightweight, simply implementing the [required interface](https://github.com/kubernetes/community/blob/master/contributors/devel/flexvolume.md#driver-invocation-model)
+The Rook Flexvolume driver will be very lightweight, simply implementing `Mount()` and `Unmount()` from the [required interface](https://github.com/kubernetes/community/blob/master/contributors/devel/flexvolume.md#driver-invocation-model)
 and then offloading the storage provider work to the Rook agent over the Unix domain socket.
+Note this means that the attach/detach controller and its centralized attaching/detaching will not be used.
+Instead, attaching will be performed as part of `Mount()` and detaching as part of `Unmount()`, as described in the next section.
+This is important because it means that the Rook Flexvolume driver and Agent do **not** need to be installed on the Kubernetes master node (control plane), which is not possible in some environments such as Google Container Engine (GKE).
```

Comment:
did you mean (controller manager) instead of control plane?

Reply:
no, i meant to use "control plane" here, but perhaps I used it incorrectly.  I wanted to convey that the master node hosts most of the control plane of the kubernetes cluster. Is it better to use say "master node" by itself without the "(control plane)"?

Reply:
this documentation makes it seem that i'm not too far off, but i'll edit the wording here to what you think is most appropriate:
https://kubernetes.io/docs/concepts/overview/components/#master-components